tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -70,7 +70,7 @@ class Meta:
         """Metadata."""
 
         model = User
-        fields = ["first_name", "last_name", "email", "stt", "roles"]
+        fields = ["id", "first_name", "last_name", "email", "stt", "roles"]
```

Comment:
We need to add `id` to `UserProfileSerializer` so that it's available on the `user` object on the frontend, and accessible when submitting the files to the backend.